### DISCUSSION TITLE
Support custom content as the stepper's icon

### DISCUSSION THREAD

**Discussion by pBouillon** [saadeghi/daisyui#2711] [Ideas / feature requests]
Hi, and thanks for the work on DaisyUI!

I was browsing the components and took a look at the stepper.
I am currently using [Bootstrap Icons](https://icons.getbootstrap.com) and was trying to embed one as the icon of the stepper but without success.

While looking for a solution, I saw the steppers that [Flowbite suggests](https://flowbite.com/docs/components/stepper) and it seems that they are offering more flexibility on various points:

- Width
- Content customization
- A "dense mode" with the icon and the text inlined

It would be great to have something similar in DaisyUI!

**Comment by saadeghi** [comment]
There's a workaround for adding custom icons:  
https://github.com/saadeghi/daisyui/discussions/2395

[steps](https://daisyui.com/components/steps/) was designed to show a numerical step with the minimum amount of HTML but you may want to use [timeline](https://daisyui.com/components/timeline/) which all parts are customizable.

Let me know if you have a question.
